src/microbe_stage/Microbe.Contact.cs

```diff
@@ -505,24 +505,20 @@ public void Kill()
             }
         }
 
-        int chunksToSpawn = Math.Max(1, HexCount / Constants.CORPSE_CHUNK_DIVISOR);
+        // queues either 1 corpse chunk or a factor of the organelles
+        int chunksToSpawn = Math.Max(1, (organelles.Count + 1) / Constants.CORPSE_CHUNK_DIVISOR -
+            (1 / Constants.CORPSE_CHUNK_DIVISOR));
```

Comment:
Updated algorithm to depend on number of organelles instead of hexes. Large organelles such as the nucleus being the only organelle would crash the game as it would try to spawn 3 organelles with only one being present